cassandane/tiny-tests/JMAPCore/echo-tls

```diff
@@ -0,0 +1,28 @@
+#!perl
+use Cassandane::Tiny;
+
+sub test_echo_tls
```

Comment:
The file name contains a dash but the test name an underscore. Please change the file name to match the test name.

Reply:
Rik usually prefers file names not have underscores because they can be hard to see in terminal output or other situations, so we use dashes instead.

Perl subroutines can't contain dashes, so we use underscores instead.

I think it would be fair to say files use dashes and subs use underscores. We have a mix of both right now but settling on one would be better.

What do you think @rsto?